node_modules/oae-mixpanel/lib/eventlisteners/invitations.js

```diff
@@ -0,0 +1,85 @@
+/*!
+ * Copyright 2015 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+var _ = require('underscore');
+
+var AuthzUtil = require('oae-authz/lib/util');
+var ResourceActions = require('oae-resource/lib/actions');
+var ResourceConstants = require('oae-resource/lib/constants').ResourceConstants;
+var TenantsAPI = require('oae-tenants');
+
+var MixpanelUtil = require('oae-mixpanel/lib/util');
+
+module.exports = function(client) {
+
+    /**
+     * One or more invitations got sent out
+     */
+    MixpanelUtil.listen(ResourceActions, ResourceConstants.events.INVITED, function(ctx, invitations, emailTokens) {
+        _.each(invitations, function(invitation) {
+            var params = MixpanelUtil.getBasicParameters(ctx);
+            params.inviteToken = emailTokens[invitation.email];
+
+            // The user who is inviting the guest
+            params.invitedBy = ctx.user().id;
```

Comment:
`invitedBy` is already part of the basic mixpanel parameters, so no need to repeat it here